mailbox/elasticsearch/src/test/java/org/apache/james/mailbox/elasticsearch/query/QueryConverterTest.java

```diff
@@ -50,14 +50,14 @@ public void allCriterionShouldBeWellConverted()throws Exception{
         assertThatJson(queryConverter.from(searchQuery, MAILBOX_UUID).toXContent(jsonBuilder(), QueryBuilder.EMPTY_PARAMS).string())
```

Comment:
Agree.
It's very important to write such tests while doing TDD but once everything is in place, you can remove the tests.
It's an usual TDD step.